packages/components/src/angle-picker-control/styles/angle-picker-control-styles.js

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
Good point! It may no longer be necessary. I originally introduced this structure to make it distinctly different from the `style.scss` during the early proposal/experimental phase.

I'm happy to keep the `style.js` files at the root level. It's the same convention I've followed in another React component library project:

https://github.com/ItsJonQ/g2/tree/master/packages/components/src/Button

Reply:
💯